tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+  "extends": "typestrict",
+  "rules": {
+    "no-console": true,
+    "no-debugger": true,
+    "mocha-avoid-only": true,
+    "prefer-const": true,
+    "no-var-keyword": true,
+    "interface-name": [true, "never-prefix"],
```

Comment:
If I get this right this means interface names should never start with an ``I`` ([docs](https://palantir.github.io/tslint/rules/interface-name/), you nevertheless prefix all interfaces on ``evm-ts`` with an ``I``. Could you explain?

Reply:
Yeah that's funny 😆 I changed my preference after using it for some time. It's also fixed already here: https://github.com/ethereum-ts/evm-ts/commit/010f4ecd9bcd6fdc67cb667eafbab1f154a02432 

So basically reasoning is following:
- types live in a different scope than other symbols so you can't confuse them.
- you can declare shape of an object in multiple ways, would all of them be prefixed by I? (ex. `type Test = { a: string };`)
- often when interface bacames union type you have insonsitent naming in the code base
- due to structural typings it doesnt matter how you define type (using `interface` keyword or defining a class etc)
- typescript internal guidelines reject doing so: https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines

Reply:
Ok, thanks for this long clarification, think taken together this is a strong and solidly based reasoning for the choice. Cool. 👍